c/meterpreter/source/extensions/stdapi/server/sys/process/process.c

```diff
@@ -381,14 +381,15 @@ DWORD request_sys_process_execute(Remote *remote, Packet *packet)
 			}
 
 			// Try to execute the process with duplicated token
-			if (!CreateProcessAsUser(pToken, NULL, commandLine, NULL, NULL, inherit, createFlags, pEnvironment, NULL, (STARTUPINFOA*)&si, &pi))
+			wchar_t *commandLine_w = met_api->string.utf8_to_wchar(commandLine);
+			if (!CreateProcessAsUserW(pToken, NULL, commandLine_w, NULL, NULL, inherit, createFlags, pEnvironment, NULL, (LPSTARTUPINFOW)&si, &pi))
 			{
 				LPCREATEPROCESSWITHTOKENW pCreateProcessWithTokenW = NULL;
 				HANDLE hAdvapi32 = NULL;
 				wchar_t * wcmdline = NULL;
 				wchar_t * wdesktop = NULL;
 				size_t size = 0;
-
+				free(commandLine_w);

```

Comment:
I think we also need to free(commandLine_w) if CreateProcessAsUserW fails. Let me try send a pull request that fixes this.